.eslintrc

```diff
@@ -6,5 +6,18 @@
   "extends": [
     "plugin:vue-libs/recommended",
     "plugin:flowtype/recommended"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "**/*.spec.js"
+      ],
+      "plugins": [
+        "jasmine"
+      ],
+      "env": {
```

Comment:
the part specific to tests can actually go into an `.eslintrc` file inside the test folder